02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+            },
+            "Should all household members be listed even if they are not in the food stamp household?": {
+                "score_sum": 0.1247161281025545,
+                "derived_questions": [
+                    "Do the roommates often cook and eat together in the shared kitchen?"
+                ]
+            },
+            "Who are mandatory HH members for food stamps?": {
+                "score_sum": 0.1190879140473805,
+                "derived_questions": [
+                    "Do the roommates often cook and eat together in the shared kitchen?"
+                ]
+            }
+        }
+    },
+    {
+        "id": 4,
+        "question": "Husband and wife consider themselves separated, but they still live in the same house. The husband lives on the 2nd floor and the wife lives on the first. They do not speak or share any food. Can the wife apply on her own?",
+        "guru_cards": [
+            "How is \"spouse\" defined for food stamp applications?"
+        ],
+        "recall": 1.0,
+        "extra_cards": 13,
+        "derived_questions": [
+            "What is the situation between the husband and wife?",
+            "Can they still be considered separated if they live in the same house?",
+            "Do they have any communication with each other?",
+            "Do they share any food or resources?",
+            "Is the wife able to apply for something on her own?"
+        ],
+        "all_retrieved_cards": {
+            "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?": {
+                "score_sum": 0.5970122321883444,
+                "derived_questions": [
+                    "Is the wife able to apply for something on her own?",
+                    "Can they still be considered separated if they live in the same house?"
+                ],
+                "entire_card": "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?\nAnswer:\nYes, if the adults with whom they are living do not exercise parental control or offer financial support. Clients who are officially emancipated usually fall under this category. If the client not entirely certain or it's close but messy we can error on the side of applying and inform the client the county will help them figure out the household if need be.\nExample: Ms. G is 17 years old. She lives with her aunt. Ms. G works and purchases and prepares her own meals. When questioned about household composition she stated that her aunt does not exercise parental control over her. The case manager certifies Ms. G as a one-person household.\nFood stamps only",
+                "summary": "Children under 18 years of age can be certified as a separate household if the adults they live with do not exercise parental control or offer financial support, and if they are officially emancipated."
```

Comment:
@aliciaben Review here

Reply:
I can't see from this snippet if more information is provided the questioner regarding who "they" is. Are they referring all adults or do they include minors. Based solely on the question above I'm gave the creating below. 

Rating: Relevance to the question
🟠 2 - The content pulled is relevant in that it talks about a condition in which individuals in the same house can be listed separately. Would be most relevant to pull an additional card that speaks to separated adults, not just children

Rating: Correctness as a summary of the Guru card
🟢 4 - Correct (100% correct)

Rating: Helpfulness

🟡 3 - Neutral (technically not wrong answer but not really helpful in the context of the question).